app/index.js

```diff
@@ -69,7 +88,17 @@ module.exports = class extends yeoman {
             showPrompts = false;
         }
 
-        if (showPrompts) {
+        let showWsdl2Rest = this.options.wsdl2rest;
+
+        var showW2EPrompts = true;
+        if (!showPrompts && showWsdl2Rest) {
+            showW2EPrompts = !(utils.isNotNull(this.options.wsdl)) &&
```

Comment:
IS it possible to use something like utils.isNull || utils.isNull ? it will avoid the double negation "!" "isNot"

Reply:
Yup!